docs/guides/on-demand.mdx

```diff
@@ -619,22 +545,19 @@ This method allows for exporting an asset imported or uploaded from Livepeer to
 
 ### Request
 
-Provide the $ASSET_ID of the asset, this can be found using the List Asset method.
+Provide the `$ASSET_ID` of the asset, this can be found using the List Asset method.
 
 ```js
 const exportAsset = async () => {
-  const response = await axios.post(
-    "https://livepeer.studio/api/asset/$ASSET_ID/export",
-    {
-      headers: {
-        Authorization: `Bearer ${process.env.LP_API_KEY}`,
-        "Content-Type": "application/json",
-      },
-      data: JSON.stringify({
-        ipfs: {},
-      }),
-    }
-  );
+  const response = await axios.post("https://livepeer.studio/api/asset/$ASSET_ID/export", {
```

Comment:
No changes needed here, but I think overall we had these `$VARIABLES` in our guide before only because we only provided bash script examples. In node, especially since the examples are generally functions like this `exportAsset`, I think any external arguments should be provided as arguments to the function instead with very clear names (likely just converting the `$` variable to a `camelCase` arg).

In this case, just as an example, it'd be:
```ts
const exportAsset = async (assetId) => {
  const response = await axios.post(`https://livepeer.studio/api/asset/${assetId}/export`, {
```

But this is a change we should make to all of the examples, so maybe rather just keep it like this for now and later we can bother about updating all the examples.